client/views/room/contextualBar/UserInfo/UserInfo.js

```diff
@@ -34,6 +35,18 @@ function UserInfo({
 
 	const timeAgo = useTimeAgo();
 
+	const customFieldsToShowSetting = useSetting('Accounts_CustomFieldsToShowInUserInfo');
+	const customFieldsToShow = [];
+	const customFieldsToShowObj = JSON.parse(customFieldsToShowSetting);
+	customFieldsToShowObj &&
+		Object.values(customFieldsToShowObj).map((value) => {
+			const role = Object.values(value);
+			const roleNameToShow = Object.keys(value);
+			const customField = {};
+			customField[roleNameToShow] = customFields[role];
+			return customFieldsToShow.push(customField);
+		});
```

Comment:
Remember that the `Array.prototype.map` method is not a callback-based for-each loop, but a functional concept:

```suggestion
	const customFieldsToShow = customFieldsToShowObj ?
		Object.values(customFieldsToShowObj).map((value) => {
			const role = Object.values(value);
			const roleNameToShow = Object.keys(value);
			const customField = {};
			customField[roleNameToShow] = customFields[role];
			return customField;
		}) : [];
```

Reply:
Thanks @tassoevan for the suggestion. I will remember this. :)